ios/Integreat.xcodeproj/project.pbxproj

```diff
@@ -484,19 +485,13 @@
 				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = 7272KE28TJ;
-				ENABLE_BITCODE = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
Is this intentional? Why?

Reply:
https://issues.tuerantuer.org/browse/NATIVE-628